### ISSUE TITLE
Pre-signup carousel content is not accessible to screen readers 

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#6719]
We need to improve the accessibility of the pre-signup carousel so that:

* Slide content is exposed to screen readers
* Slides are navigable by key clicks
* It’s obvious to people using screen readers which slide they’re on and how many total slides there are. This should not be communicated by indicator dots alone.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-4642)
┆Reporter: Lesley Norton
